Amigolden/src/app/pages/messages/messages.page.ts

```diff
@@ -1,128 +1,148 @@
-import { Identity } from 'src/app/services/identity/identity.service';
-import { MessagesService } from 'src/app/services/endpoints/messages.service';
-import { HubServiceBase } from 'src/app/services/hubs/hub-base.service';
-import { Router, ActivatedRoute } from '@angular/router';
-import { ListConfiguration, PagingInfo } from 'src/app/components/list-base/data/list-configuration';
-import { map } from 'rxjs/operators';
-import { ListBaseComponent } from 'src/app/components/list-base/list-base.component';
-import { PageBase } from '../detail-page-base';
+import { NgZone, ViewChild, Component, OnInit } from "@angular/core";
+import moment from "moment";
+import { EditOptions } from "../../models/edit-options";
+import { Conversation } from "../../models/conversation";
+import { Message } from "../../models/message";
+import {
+  ODataDynamicFilterBuilder,
+  ODataPropertyPath,
+} from "../../models/odata/filter/ts-odata-dynamic-filter";
+import { Identity } from "src/app/services/identity/identity.service";
+import { MessagesService } from "src/app/services/endpoints/messages.service";
+import { HubServiceBase } from "src/app/services/hubs/hub-base.service";
+import { Router, ActivatedRoute } from "@angular/router";
+import {
+  ListConfiguration,
+  PagingInfo,
+} from "src/app/components/list-base/data/list-configuration";
+import { map } from "rxjs/operators";
+import { ListBaseComponent } from "src/app/components/list-base/list-base.component";
+import { PageBase } from "../detail-page-base";
 
 @Component({
-  selector: 'app-messages',
-  templateUrl: './messages.page.html',
-  styleUrls: ['./messages.page.scss'],
-  providers: [ HubServiceBase ]
+  selector: "app-messages",
+  templateUrl: "./messages.page.html",
+  styleUrls: ["./messages.page.scss"],
+  providers: [HubServiceBase],
 })
 export class MessagesPage extends PageBase implements OnInit {
+  // @ViewChild('content') content: any;
 
-    // @ViewChild('content') content: any;
+  @ViewChild("pageListBase", undefined) pageListBase: ListBaseComponent<
+    Message
+  >;
 
-    @ViewChild('pageListBase', undefined) pageListBase: ListBaseComponent<Message>;
+  pageSize = 20;
+  pageNumber = 1;
 
-    pageSize = 20;
-    pageNumber = 1;
+  filterByConversationId: string;
 
-    filterByConversationId: string;
+  messageText: string;
 
-    messageText: string;
+  canSendMessage: boolean;
 
-    canSendMessage: boolean;
+  editOptions = new EditOptions();
 
-    editOptions = new EditOptions();
-
-    public config = new ListConfiguration<Message>(
-      (pagingInfo: PagingInfo) =>
-      this.baseProvider.getDynamicList(this.filterByConversationId,
-        pagingInfo.pageSize, pagingInfo.pageNumber).pipe(map(message => {
+  public config = new ListConfiguration<Message>(
+    (pagingInfo: PagingInfo) =>
+      this.baseProvider
+        .getDynamicList(
+          this.filterByConversationId,
+          pagingInfo.pageSize,
+          pagingInfo.pageNumber
+        )
```

Comment:
this file shouldn't have changes remove it from the commit

Reply:
removed